### PR TITLE
commander: add force arm CLI option (commander arm -f)

### PR DESCRIPTION
**Pull request by bkueng**
This allows to bypass preflight and prearm checks on the shell via `commander arm -f`.

During development there are regular cases where I just want to arm the
board/vehicle no matter what, and the preflight checks are guaranteed to
fail (e.g. sensors uncalibrated, inconsistent, powered via USB, etc.).
Allowing an easy and quick way to arm (assuming you know what you are
doing) helps to speed up development considerably and is less frustrating.

See individual commit messages for further details.

### CODE REVIEW

File: src/modules/commander/Commander.cpp [hunk 1]
```diff
@@ -201,6 +201,7 @@ void print_status();
 bool shutdown_if_allowed();
 
 transition_result_t arm_disarm(bool arm, orb_advert_t *mavlink_log_pub, const char *armedBy);
+transition_result_t arm_disarm(bool arm, bool run_preflight_checks, orb_advert_t *mavlink_log_pub, const char *armedBy);
```
Comment (julianoes) [inline src/modules/commander/Commander.cpp]: Can we kill off the old function declaration?

File: src/modules/commander/Commander.cpp [hunk 2]
```diff
@@ -516,7 +523,7 @@ void usage(const char *reason)
 		PX4_INFO("%s", reason);
 	}
 
-	PX4_INFO("usage: commander {start|stop|status|calibrate|check|arm|disarm|takeoff|land|transition|mode}\n");
+	PX4_INFO("usage: commander {start|stop|status|calibrate|check|arm [-f]|disarm|takeoff|land|transition|mode}\n");
```
Comment (julianoes) [inline src/modules/commander/Commander.cpp]: Is there no cli docs for commander where this needs to be added as well?

File: src/modules/commander/Commander.cpp (hunk 1 above)
Comment (bkueng) [reply src/modules/commander/Commander.cpp]: Done, that's a rebase fail.

File: src/modules/commander/Commander.cpp (hunk 2 above)
Comment (bkueng) [reply src/modules/commander/Commander.cpp]: No, but I can add them.

Comment (bkueng) [reply src/modules/commander/Commander.cpp]: Done: https://github.com/PX4/Firmware/pull/12715/commits/802ba74b8a4ae4ba28f2747826366f486a54740c

Comment (julianoes) [reply src/modules/commander/Commander.cpp]: Nice!

**Comment by jlecoeur** [comment]
@bkueng good idea. 

How about the same option on the mavlink interface?
In https://mavlink.io/en/messages/common.html#MAV_CMD_COMPONENT_ARM_DISARM that would look like:
param 3: enforce preflight checks.
0: only arm if preflight checks pass
26247: enforce arming even if preflight checks do not pass

**Comment by bkueng** [comment]
> How about the same option on the mavlink interface?

Yes we can do that, but I think the bigger question is around the UI: regular users should not do this, and probably not even see the option to allow them to force-arm.

**Pull request merged**